.dockerignore

```diff
@@ -1,8 +1,11 @@
+.git
+output/*
+build.sh
 _site
 .vagrant
 Vagrantfile
 git
 pkg/*
 provider/provider
 summon
-junit.xml
\ No newline at end of file
+junit.xml
```

Comment:
@dustinmm80 it all LGTM but i suppose the junit.xml isn't necessary here since everything is in output/*